src/tests/ftest/interoperability/updown_grade.py

```diff
@@ -9,24 +9,25 @@
 import base64
 import time
 
-from general_utils import get_random_bytes, pcmd, run_pcmd
 from pydaos.raw import DaosApiError
+from general_utils import get_random_bytes, pcmd, run_pcmd
 from agent_utils import include_local_host
 from command_utils_base import CommandFailure
 from ior_test_base import IorTestBase
 
-# pylint: disable=global-variable-not-assigned,global-statement
+# pyint: disable=global-variable-not-assigned,global-statement
```

Comment:
Typo? `pyint -> pylint`

Reply:
Thanks for the comment, please see the updates.